hw1/02-palindrome.html

```diff
@@ -15,11 +16,11 @@
         <h1>Exercise 02 - Palindrome</h1>
         <div>
           <label>Enter a positive number:</label>
```

Comment:
If you Wave the web page, then this is an accessibility error. The label is not associated with an input label, meaning you should add the for="palindrome" as one of the attributes of the label element.

Reply:
Really good observation.  Thanks for the heads up!